lib/workers/repository/update/pr/changelog/source.ts

```diff
@@ -139,20 +141,9 @@ export abstract class ChangeLogSource {
           changes: [],
           compare: {},
         };
-        const prevHead = await this.getRef(
-          version,
-          packageName,
-          prev,
-          apiBaseUrl,
-          repository
-        );
-        const nextHead = await this.getRef(
-          version,
-          packageName,
-          next,
-          apiBaseUrl,
-          repository
-        );
+        const tags = await getTags();
```

Comment:
wonder why we can do it once before the loop?

Reply:
If we have cached results, we don't even call it